.env.template

```diff
@@ -33,11 +32,7 @@ MODELS=`[
       "name": "meta-llama/Llama-2-70b-chat-hf",
       "description": "The latest and biggest model from Meta, fine-tuned for chat.",
       "websiteUrl": "https://ai.meta.com/llama/",
-      "userMessageToken": "",
-      "userMessageEndToken": " [/INST] ",
-      "assistantMessageToken": "",
-      "assistantMessageEndToken": " </s><s>[INST] ",
-      "preprompt": " ",
+      "preprompt": "",
       "chatPromptTemplate" : "<s>[INST] <<SYS>>\n{{preprompt}}\n<</SYS>>\n\n{{#each messages}}{{#ifUser}}{{content}} [/INST] {{/ifUser}}{{#ifAssistant}}{{content}} </s><s>[INST] {{/ifAssistant}}{{/each}}",
```

Comment:
can't remove it for llama-2-70b-chat-hf because the model is gated and so is the chat template, too bad haha